reindex.go

```diff
@@ -333,6 +333,8 @@ func ReindexPrefix(db *ICATConnection, es *ESConnection, prefix, irodsZone strin
 		return err
 	}
 
+	rb() // Roll back tx as early as possible
```

Comment:
Because processing the deletions doesn't require the ability to make more DB queries, we can roll back here before doing that.